packages/datadog_flutter_plugin/lib/src/datadog_configuration.dart

```diff
@@ -179,15 +181,25 @@ class RumConfiguration {
   ///
   /// The sampling rate must be a value between `0.0` and `100.0`. A value of
   /// `0.0` means no RUM events will be sent, `100.0` means all sessions will be
-  /// kept
+  /// sent
   ///
   /// Defaults to `100.0`.
   double sampleRate;
```

Comment:
I think it is better to rename this one to `sessionSamplingRate` for the better clarity (but it is already part of public API though).

Reply:
Yeah, I'll deprecate it and map it to a new variable for at least one release.